Modules/Microsoft365DSC/DSCResources/MSFT_O365User/MSFT_O365User.schema.mof

```diff
@@ -26,8 +26,7 @@ class MSFT_O365User : OMI_BaseResource
     [Write, Description("Present ensures the user exists, absent ensures it is removed"), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
     [Write, Description("Credentials of the Exchange Global Admin"), EmbeddedInstance("MSFT_Credential")] string Credential;
     [Write, Description("Id of the Azure Active Directory application to authenticate with.")] String ApplicationId;
-    [Write, Description("Id of the Azure Active Directory tenant used for authentication.")] String TenantId;
+    [Write, Description("Name of the Azure Active Directory tenant used for authentication. Format contoso.onmicrosoft.com")] String TenantId;
+    [Write, Description("Secret of the Azure Active Directory tenant used for authentication.")] String ApplicationSecret;
```

Comment:
Shouldn't this be Azure Active Directory application instead of tenant?

Reply:
Same as above.

Reply:
Done